fastlane/lanes/localization.rb

```diff
@@ -223,50 +266,80 @@
   #
   desc 'Downloads the localized metadata (for App Store Connect) from GlotPress for the WordPress app'
   lane :download_wordpress_localized_app_store_metadata do
-    # @FIXME: Replace this whole lane with a call to the future replacement of `gp_downloadmetadata` once it's implemented in the release-toolkit (see paaHJt-31O-p2).
-
-    # No need to `cd` into `fastlane` because of how Fastlane manages its paths internally.
-    sh './download_metadata.swift wordpress'
+    metadata_directory = File.join(PROJECT_ROOT_FOLDER, 'fastlane', 'metadata')
 
-    # @TODO: Make the `fastlane/metadata/en-US/release_notes.txt` path be the source of truth for the original copies in the future.
+    # FIXME: We should make the `fastlane/metadata/default/release_notes.txt` path be the source of truth for the original copies in the future.
     # (will require changes in the `update_appstore_strings` lane, the Release Scenario, the MC tool to generate the announcement post…)
     #
-    # In the meantime, since GlotPress doesn't have the `en-US` notes because those are the ones used as originals, just copy the file to the right place for `deliver` to find
-    metadata_directory = File.join(PROJECT_ROOT_FOLDER, 'fastlane', 'metadata')
+    # In the meantime, just copy the file to the right place for `deliver` to find, for the `default` pseudo-locale which is used as fallback
     release_notes_source = File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Resources', 'release_notes.txt')
-    FileUtils.cp(release_notes_source, File.join(metadata_directory, 'en-US', 'release_notes.txt'))
-
-    metadata_path = File.join(metadata_directory, '**', '*.txt')
-    git_add(path: metadata_path, shell_escape: true)
```

Comment:
@mokagio  ⚠️ Notice how you used `shell_escape: true` before, but I changed it to `shell_escape: false` [here](https://github.com/wordpress-mobile/WordPress-iOS/pull/18439/files#diff-ad77c85d7eb3034dec610211cb1747f7cac134e425448b1dcc246d71df3e8639R339).

I think `shell_escape: false` is the right choice, because we pass a parameter looking like `fastlane/metadata/**/*.txt` so we **want** this to be passed **unescaped** to the `git add` shell command, so that the shell interprets those `*` as glob patterns, and matches them all. While `shell_escape: true` would lead to `git_commit` adding shell escapes to the paths and transforming them to `fastlane/metadata/\*\*/\*.txt` and trying to match files with literal `*` characters in their name.

That being said, I'm a bit surprised that `git_add(path: metadata_path, shell_escape: true)` worked before that PR. I think we were just lucky because that `git_add` would then not find any file with those literal `*` characters and thus add nothing (`git add` would be a no-op there… instead of erroring?!), and then `git_commit` would take care of doing the staging and adding of the files to git (which only works with files that were already known to git and wouldn't work for brand new files, but we were probably lucky to never encounter that use case in practice, since that code was introduced recently?)

Reply:
> That being said, I'm a bit surprised that `git_add(path: metadata_path, shell_escape: true)` worked before that PR. I think we were just lucky because that `git_add` would then not find any file with those literal `*` characters and thus add nothing (`git add` would be a no-op there… instead of erroring?!), and then `git_commit` would take care of doing the staging and adding of the files to git (which only works with files that were already known to git and wouldn't work for brand new files, but we were probably lucky to never encounter that use case in practice, since that code was introduced recently?)

🤔 

I added that `git_add(path: ..., shell_escape: true)` via 18f3bf73d1314f58321f3236a3a6a3aabef5be2b, but I can't remember why exactly. I added good details in the commit message about why the change was necessary, but not about why I went out of my way to explicitly pass the `shell_escape` option, set to the default value 🤔 

However, `git add` with `shell_escape: true` is _not_ a no-op. It indeed adds any file matching the pattern... even if escaped 🤔 

<img width="979" alt="image" src="https://user-images.githubusercontent.com/1218433/165236300-40f18eea-ca94-441b-968f-cea40e62c16d.png">

But, that might be because of Git itself:

<img width="850" alt="image" src="https://user-images.githubusercontent.com/1218433/165236443-754cf33c-b2ea-43a3-8cd7-b81d402c0986.png">

I'm running out of time in my day to explore this in more details. I'm happy with `false`, which is after all what the documentation recommends to use in this case:

```
+--------------+------------------------+------------+---------+
|                       git_add Options                        |
+--------------+------------------------+------------+---------+
| Key          | Description            | Env Var(s) | Default |
+--------------+------------------------+------------+---------+
| path         | The file(s) and        |            |         |
|              | path(s) you want to    |            |         |
|              | add                    |            |         |
| shell_escape | Shell escapes paths    |            | true    |
|              | (set to false if       |            |         |
|              | using wildcards or     |            |         |
|              | manually escaping      |            |         |
|              | spaces in :path)       |            |         |
```

(Maybe that's what happened! I read the docs, saw we'd need the option, but used the wrong value somehow 🤷‍♂️ )